torchmdnet/utils.py

```diff
@@ -5,6 +5,35 @@
 from os.path import dirname, join, exists
 from pytorch_lightning.utilities import rank_zero_warn
 
+# fmt: off
```

Comment:
Maybe you could include a comment with the source of numbers.

Reply:
Sure, I took the comment on the masses now from ASE.